alliance/manage.py

```diff
@@ -3,6 +3,6 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "alliance.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "alliance.config.settings.base")
```

Comment:
I think that this should be a bit two fold:

    django_settings_module = os.getenv('DJANGO_SETTINGS_MODULE', 'alliance.config.settings.base')
    os.environ.setdefault('DJANGO_SETTINGS_MODULE', django_settings_module)

This would give the user the ability to have an environment variable to specify which settings file to use (`dev`, `prod`, `test`).